src/App/Application.cpp

```diff
@@ -2240,52 +2235,52 @@ void parseProgramOptions(int ac, char ** av, const string& exe, variables_map& v
     boost::program_options::options_description config("Configuration");
     config.add_options()
     ("write-log,l", descr.str().c_str())
-    ("log-file", value<string>(), "Unlike --write-log this allows logging to an arbitrary file")
-    ("user-cfg,u", value<string>(),"User config file to load/save user settings")
-    ("system-cfg,s", value<string>(),"System config file to load/save system settings")
-    ("run-test,t", value<string>()->implicit_value(""),"Run a given test case (use 0 (zero) to run all tests). If no argument is provided then return list of all available tests.")
-    ("run-open,r", value<string>()->implicit_value(""),"Run a given test case (use 0 (zero) to run all tests). If no argument is provided then return list of all available tests.  Keeps UI open after test(s) complete.")
-    ("module-path,M", value< vector<string> >()->composing(),"Additional module paths")
-    ("macro-path,E", value< vector<string> >()->composing(),"Additional macro paths")
-    ("python-path,P", value< vector<string> >()->composing(),"Additional python paths")
-    ("disable-addon", value< vector<string> >()->composing(),"Disable a given addon.")
+    ("log-file", boost::program_options::value<std::string>(), "Unlike --write-log this allows logging to an arbitrary file")
+    ("user-cfg,u", boost::program_options::value<std::string>(),"User config file to load/save user settings")
+    ("system-cfg,s", boost::program_options::value<std::string>(),"System config file to load/save system settings")
+    ("run-test,t", boost::program_options::value<std::string>()->implicit_value(""),"Run a given test case (use 0 (zero) to run all tests). If no argument is provided then return list of all available tests.")
+    ("run-open,r", boost::program_options::value<std::string>()->implicit_value(""),"Run a given test case (use 0 (zero) to run all tests). If no argument is provided then return list of all available tests.  Keeps UI open after test(s) complete.")
+    ("module-path,M", boost::program_options::value< std::vector<std::string> >()->composing(),"Additional module paths")

```

Comment:
Just a note, if you find any spaces between end angled brackets like `> >` on lines you change, feel free to remove them at the same time as they are a relic from the past when c++ couldn't understand the difference between `std::vector<std::vector<int>>` and `std::vector<std::vector<int` + the operator `>>`.

In this case:
```suggestion
    ("module-path,M", boost::program_options::value<std::vector<std::string>>()->composing(),"Additional module paths")
```

No need to go back and fix that though